sharedbuffers/mapped_struct.py

```diff
@@ -2003,9 +2026,10 @@ def _subset(self, seq, strict_subset):
                 dcode2, objlen2, itemsize2, offset2, _struct2 = pfset.objlist._metadata()
                 for i in xrange(xlen):
                     val = self.objlist._c_getitem(i, dcode, objlen, itemsize, offset, _struct, None)
-                    j = pfset._search_key(val, dcode, offset, j, seqlen - j, j)
-                    if pfset.objlist._c_getitem(j, dcode2, objlen2, itemsize2, offset2, _struct2, None) != val:
+                    tmp_idx = pfset._search_key(val, dcode, offset, j, seqlen - j, j, True)
+                    if not tmp_idx < seqlen - j:
```

Comment:
I prefer `tmp_idx >= seqlen - j`. Other code uses that IIRC.

Reply:
Sure.